src/utilities/workflowAnnotationUtils.test.ts

```diff
@@ -16,5 +16,18 @@ describe('WorkflowAnnotationUtils', () => {
             cleanLabel('Workflow Name / With Slashes / And Spaces')
          ).toEqual('Workflow_Name_-_With_Slashes_-_And_Spaces')
       })
+      it('should return a blank string when regex fails (https://github.com/Azure/k8s-deploy/issues/266)', () => {
+         const label = '持续部署'
+         expect(cleanLabel(label)).toEqual('')
```

Comment:
this doesn't seem ideal. we should be able to label it somehow instead of just getting a blank string back

Reply:
I can change the default to something like "regexfailure" or "workflowfilename"?

Reply:
changed it to "github-workflow-file"